textbook/03/1/Operations.ipynb

```diff
@@ -4,16 +4,470 @@
    "cell_type": "markdown",
```

Comment:
<p>Thirteen and five  -&gt;  thirteen and five</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>